tests/common.rs

```diff
@@ -35,13 +38,34 @@ pub async fn select_client_type(s: &str) -> Result<Client, error::NewSessionErro
                     }
             });
             caps.insert("goog:chromeOptions".to_string(), opts.clone());
-
-            Client::with_capabilities("http://localhost:9515", caps).await
+            caps
         }
         browser => unimplemented!("unsupported browser backend {}", browser),
     }
 }
 
+#[cfg(feature = "rustls-tls")]
+pub async fn rustls_type(s: &str) -> Result<Client<hyper_rustls::HttpsConnector<hyper::client::HttpConnector>>, error::NewSessionError> {
```

Comment:
Should these be called `rustls_client`/openssl_client` instead? The `_type` suffix is weird to me here.